### ISSUE TITLE
Abstract in multiple languages

### ISSUE THREAD

**Issue by jomigo96** [ekspek/ist-thesis#13]
For thesis written in english by portuguese students, it is usual to have the abstract written in both languages.
The `tabstract` environment currently uses the document language. Suggest to take an optional parameter to configure the language for that environment only, which can default to the document main language.
The following should be possible:

```tex
\begin{tabstract}{english}{Keyord1, Keyword2}
    My abstract in english.
\end{tabstract}

\begin{tabstract}{portuguese}{Palavra-chave 1, Palavra-chave 2}
    O meu resumo em português.
\end{tabstract}
```

**Comment by ekspek** [comment]
It's a good point. I'll see what I can do, but it might take a bit longer than expected because we might have to tinker with ```babel``` and ```polyglossia```'s second language feature for any macros used inside the foreign language's abstract.

In any case, I'm thinking putting the language as the second argument as it's easier to handle when omitted (limited options). Either that or making a new environment altogether for either language, but that's more of a last resort.

**Comment by ekspek** [comment]
Issue is being dealt in a separate branch. Commit f37f33cbfdca52ed9f7c497c8ed7c92768c8a91d just added a ```fabstract``` environment, short for _foreign abstract_. It detects the document's default language and typesets the abstract in the alternative language.

It doesn't implement any ```babel``` or ```polyglossia``` niceties yet, but it should suffice while I sort out a better solution.

**Comment by ekspek** [comment]
Okay so in commit d9fd80d1cbf4a521fc43b9907e27076375a6f001 the ```fabstract``` kind of works like how you described right now, with ```babel``` and ```polyglossia``` implemented. The problem is the second argument (language) isn't optional, and from testing with ```cleveref```, changing the language via ```polyglossia``` (haven't tested ```babel```) isn't doing much.

I'll leaving this as it is for a while, as it's working for most uses, but I'll leave the issue open and mess around with it a bit more later on.

**Comment by ekspek** [comment]
So due to the way LaTeX handles arguments for environments I've decided to settle on the ```fabstract``` environment. It now works with ```babel``` since commit c871c6cc189f42a2bdecdbe5fc66ef3325ee70f7; I've removed ```polyglossia``` entirely from the class because it doesn't integrate very well with other packages (check [this Stack Exchange answer](https://tex.stackexchange.com/a/432407) as an example) for no real advantage. Maybe in the future if it gets fixed or there's a good reason (like international students with foreign alphabet words like Greek or Cyrillic having trouble with ```babel```) I'll re-implement it.

```fabstract``` was added to the documentation and sample thesis.

Also, as a side note from my last comment, ```cleveref``` wasn't working because it doesn't have a Portuguese translation. Take note if you're planning on using it.
